### PR TITLE
feat: rss for blog

### PR DESCRIPTION
**Pull request by hyoban**

### CODE REVIEW

File: server/routes/blog/rss.xml.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      description: doc.description,
+      author: doc.authors[0]?.name,
+      categories: [doc.category]
+    })
+  })
+
+  const feedString = feed.xml({ indent: true })
+
+  event.node.res.setHeader('content-type', 'text/xml')
+  event.node.res.end(feedString)
```
Comment (manniL) [inline server/routes/blog/rss.xml.ts:36]: In Nitro and H3, the best is not to rely on `.node` but instead use helper methods 😋 
You can also return the result straight away.

```suggestion
  appendHeader(event, 'Content-Type', 'text/xml')
  return feedString
```

**Review by manniL — CHANGES_REQUESTED** [review]
LGTM besides not relying on `.node`.

One could even go further using `$fetch` to fetch the content for each post, see. e.g @harlan-zw's website @ https://github.com/harlan-zw/harlanzw.com/blob/main/server/util/rss.ts#L6
But that is not always necessary.

File: server/routes/blog/rss.xml.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import * as cheerio from 'cheerio'
+import { Feed } from 'feed'
+
+const BaseUrl = 'https://nuxt.com'
+const SiteUrl = `${BaseUrl}/blog`
+const SiteName = 'The Nuxt Blog'
+const SiteDescription =
+  'Read the latest news about all Nuxt solutions, from framework announcements to integration tutorials.'
+const SiteLanguage = 'en'
+const Copyright = `Copyright © 2016-${ new Date().getFullYear() } Nuxt - MIT License`
```
Comment (manniL) [inline server/routes/blog/rss.xml.ts:11]: Not sure if the Blog contents are licensed under MIT though 🤔

Comment (hyoban) [reply server/routes/blog/rss.xml.ts:11]: Change to `Copyright © 2016-${ new Date().getFullYear() } Nuxt All Rights Reserved`?

Comment (manniL) [reply server/routes/blog/rss.xml.ts:11]: That's fine for me, yes 👍🏻

**Comment by Atinux** [comment]
I simplified to avoid using cheerio and adding the content, the user can click on the rss to read on the website, otherwise it's a nightmare with the code blocks and everything.

**Pull request merged**